db/migrate/20240517085142_change_lesson_id_column_type_and_copy_values.rb

```diff
@@ -0,0 +1,51 @@
+class ChangeLessonIdColumnTypeAndCopyValues < ActiveRecord::Migration[7.0]
+  def up
+    # rubocop:disable Rails/SkipsModelValidations
+    # rubocop:disable Rails/DangerousColumnNames
+    add_column(:lessons, :old_id, :integer)
+    add_column(:grades, :old_lesson_id, :integer)
+    add_column(:absences, :old_lesson_id, :integer)
+
+    Lesson.update_all('old_id = id')
+    Grade.update_all('old_lesson_id = lesson_id')
+    Absence.update_all('old_lesson_id = lesson_id')
+
+    remove_column(:lessons, :id)
+    rename_column(:lessons, :uid, :id)
+    execute 'ALTER TABLE lessons ADD PRIMARY KEY (id);'
+    remove_column(:grades, :lesson_id)
+    remove_column(:absences, :lesson_id)
+
+    add_column(:grades, :lesson_id, :uuid, null: false, default: 'uuid_generate_v4()')
```

Comment:
Why can't we rename the `lesson_uid` column to `lesson_id`? It will be way faster and less dangerous than generating 1.7 Million rows with new uuids that will be overwritten right away